adapters/trustx/usersync.go

```diff
@@ -0,0 +1,12 @@
+package trustx
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewTrustxSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("trustx", 0, temp, adapters.SyncTypeRedirect)
```

Comment:
I don't see TrustX as registered with the IAB, so a 0 vendor id makes sense here. BIDSWITCH is registered with id 128 and it seems their servers are being used. I'd just like to verify with you that there should be no vendor id for Trustx.

Reply:
Thank you, we will use 686 - the same as in TheMediaGrid

Reply:
Sounds good. I'm not sure of the legal relationship between the entities within the context of GDPR, so I'm fine with whatever you say you need to do here.